lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -7,18 +7,7 @@
 import java.io.PrintWriter;
 import java.io.UnsupportedEncodingException;
 import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Please don't use wildcard imports

Reply:
Fixed here. I need to fix this in intellij too!

Reply:
this is weird. I see that you fixed this in one of the commits, but in the overall PR it still shows it as a wildcard import.